poetry.lock

```diff
@@ -33,6 +33,17 @@ tests = ["attrs[tests-no-zope]", "zope-interface"]
 tests-mypy = ["mypy (>=1.6)", "pytest-mypy-plugins"]
 tests-no-zope = ["attrs[tests-mypy]", "cloudpickle", "hypothesis", "pympler", "pytest (>=4.3.0)", "pytest-xdist[psutil]"]
 
+[[package]]
```

Comment:
Are all these updates really needed?

Reply:
This commit didn't lock the files
https://github.com/mistralai/mistral-common/commit/d991efa48285b80cbe8c3b1b84043826732c60e4